content/en/network_monitoring/devices/snmp_metrics.md

```diff
@@ -172,6 +172,84 @@ snmp_listener:
 {{% /tab %}}
 {{< /tabs >}}
 
+#### Ping
```

Comment:
Four #'s will make this a subheading under "Autodiscovery", which I don't think is what you want.